index.js

```diff
@@ -26,11 +24,17 @@ function parseDatesFromArgv() {
     const startDate = process.argv[2];
     const endDate = process.argv[3];
 
-    const startMoment = moment.tz(startDate, timeZone).startOf('day');
+    const startMoment = DateTime.fromISO(startDate, {
```

Comment:
It seems like it would be worth creating a function createUtcLuxonFromISO(), to avoid duplicating the same few lines of code at least 3 times.

Reply:
Done